package.json

```diff
@@ -323,6 +323,7 @@
     "@graphql-codegen/cli": "^1.9.1",
     "@graphql-codegen/fragment-matcher": "^1.9.0",
     "@graphql-codegen/introspection": "^1.9.0",
+    "@slack/bolt": "^3.17.1",
```

Comment:
If this is no longer being used by the script to generate images, please delete it